pointerevents/pointerevent_gotpointercapture_before_first_pointerevent-manual.html

```diff
@@ -39,8 +39,8 @@
                             check_PointerEvent(event);
 
                             // TA: 10.2
-                            assert_true(isWaiting, "gotpointercapture must be fired asynchronously");
-                            isWaiting = false;
+                            assert_equals(waitState, 3, "gotpointercapture must be fired after pointerover and pointerenter all asynchronously");
+                            waitState = 0;
```

Comment:
Perhaps split this into 3 booleans: receivedPointerDown, receivedPointerOver and receivedPointerEnter to make the transitions clear.